app/config.yml

```diff
@@ -0,0 +1 @@
+working_path: /pointing/to/path/where/workspaces/are/stored
```

Comment:
could you add a comment around working_path 
```
# place where all the created workspaces are stored
```
please default it to something working so that the creation does not fail when users download the application and try it.
```
working_path: /tmp/
```